### ISSUE TITLE
Accessibility Issue Adaptive Card - Input fields and visual labels are not associated

### ISSUE THREAD

**Issue by Bradrajkumar** [microsoft/AdaptiveCards#8019]
**Issue Description : -**

If a relationship between elements is expressed with visual cues, it also needs to be programmatically determined. This enables assistive technology to communicate the relationship to users. Content may otherwise seem unstructured or disorganised.
There are form fields that are not associated with their visual labels. The purpose of the form fields may be unclear to users.

**Success Criteria : -**

Any relationship presented through visual cues must also use the appropriate semantics. Make sure that form fields are associated with their visual labels. In this case, wrap the visual label such as “Please give us some more…”, “First name”, “Last name” and “Email address” within the element. For example:

<label for="i1"...>
<span ...>Please give us some more details to help us respond
<span...>Required

<textarea id="i1" aria-required="true" ...></textarea>
Avoid using multiple labelling techniques on the same form field. For example, using and aria-label together will cause the aria-label to overwrite the . As well, using aria-labelledby will overwrite any other labelling techniques.

![17_1](https://user-images.githubusercontent.com/34365788/200261792-91a57bf1-9d96-4e1a-a172-5bf27ded009a.png)
![17_2](https://user-images.githubusercontent.com/34365788/200261779-6077e809-76cf-45b5-a3c3-917bd476bc33.png)

**Steps to reproduce :-**

Observe the visual labels for the form fields.
Inspect the form fields and review the DOM.
Observe that the visual labels are not associated with the form fields.
Observe that there is not an alternative labelling method that serves the same purpose
Webchat Version :- 4.13.0

**Link to raised Issue :-**

There is an issue thread raised to Microsoft Bot framework - webchat Team and as per their Comments it is an adaptive card Issue.
 Please check this link for detailed information :- https://github.com/microsoft/BotFramework-WebChat/issues/4468.

**Comment by JeanRoca** [comment]
Would using associatedInputs in [Action.Submit ](https://adaptivecards.io/explorer/Action.Submit.html)not work for this?

<img width="933" alt="image" src="https://user-images.githubusercontent.com/33772802/201782698-f99b01d1-c17a-4711-85f3-88881e6e5764.png">

**Comment by Bradrajkumar** [comment]
Hi @paulcam206 , @JeanRoca ,

please reopen this issue as workaround which you have given me as an input is not working as expected from accessibility point of view.
is it possible to use single associated id for label and form control instead of multiple ids ??
please recheck this link for more information provided by webchat team :- https://github.com/microsoft/BotFramework-WebChat/issues/4468

**Comment by JeanRoca** [comment]
> Hi @paulcam206 , @JeanRoca ,
> 
> please reopen this issue as workaround which you have given me as an input is not working as expected from accessibility point of view. is it possible to use single associated id for label and form control instead of multiple ids ?? please recheck this link for more information provided by webchat team :- [microsoft/BotFramework-WebChat#4468](https://github.com/microsoft/BotFramework-WebChat/issues/4468)

@Bradrajkumar which part is not working from an accessibility point of view? Please provide more details on this.

@anna-dingler FYI

**Comment by compulim** [comment]
@JeanRoca you can refer to https://github.com/microsoft/BotFramework-WebChat/issues/4468 on why the DOM tree is incorrect on accessibility perspective. Although this issue was originally filed to Web Chat repo, I repro-ed this bug on Adaptive Cards Designer.

For further information, please talk to the customer directly @Bradrajkumar.
